tokio/src/runtime/threadpool/builder.rs

```diff
@@ -58,6 +58,8 @@ pub struct Builder {
 
     /// The clock to use
     clock: Clock,
+
+    around_worker: Option<Arc<dyn Fn(&threadpool::Worker) + Send + Sync + 'static>>,
```

Comment:
Nit: every other field has a doc comment, and this is one of the less self-explanatory fields IMO